packages/iTwinUI-react/src/core/Table/columns/actionColumn.tsx

```diff
@@ -99,6 +101,8 @@ export const ActionColumn = <T extends Record<string, unknown>>({
           menuItems={headerCheckBoxes}
           onHide={() => setIsOpen(false)}
           onShow={() => setIsOpen(true)}
+          className={className}
+          style={{ maxHeight: '300px', overflow: 'scroll', ...style }} // Default style (max height) that can be overridden/unset
```

Comment:
Are the default values okay for when the `DropdownMenu` when no custom style is provided?

Reply:
**[OUTDATED]**